app/scripts/views/mixins/password-mixin.js

```diff
@@ -12,6 +12,7 @@ define(function (require, exports, module) {
 
   module.exports = {
     events: {
+      'focus input.password': 'onPasswordFocus',
```

Comment:
@shane-tomlinson Unsure how to handle this. I thought of the following scenarios:
1. Try to run the same function when the any input field is focused, but this fails if the user clicks outside the input fields, clicks on ToS/PP and hits back
2. Onpageload? I am not sure how to get a handler to fire for this event
Could you let me know?